### ISSUE TITLE
Clarifications on sub-setting in grid space

### ISSUE THREAD

**Issue by jerstlouis** [opengeospatial/ogc_api_coverages#106]
What defines the axis name of grid space? Is it implied that it is CRS:1, and the axes should always be named `i` and `j`, where the upper bound for `j` corresponds to the lower bound of `lat` in CRS84?
Is that CRS always available?

Should _OGC API - Coverages_ conformance classes have any requirements and/or clarifications about this?

**Comment by pebau** [comment]
> What defines the axis name of grid space?

`<GridLimits srsName="http://www.opengis.net/def/crs/OGC/0/Index2D" axisLabels="i j">`

...pick whatever axis names you like there.

**Comment by jerstlouis** [comment]
@pebau Right, I see that http://www.opengis.net/def/crs/OGC/0/Index2D does have the `<axisAbbrev>i</axisAbbrev>` and `<axisAbbrev>j</axisAbbrev>` so there is no confusion here.

>...pick whatever axis names you like there.

The specifications are saying at the moment though that those names are defined by the CRS, so the server cannot just arbitrarily pick whatever names it likes.

I think we need to clarify the [REQ_cov-subset-definition](https://github.com/opengeospatial/ogc_api_coverages/blob/master/standard/requirements/coverage-subset/REQ_cov-subset-definition.adoc) with the same type of requirement as the [requirement D for scaling](https://github.com/opengeospatial/ogc_api_coverages/blob/master/standard/requirements/coverage-scaling/REQ_cov-scaling-definition.adoc) which clearly states where those axis names are defined per the CRS (in addition to being listed in the DomainSet).

But the main point here I think is we need to explicitly state whether support for indexing in grid space is a requirement for scaling and/or subsetting, and perhaps as well provide additional requirements about whether grid axes should be included in the [domainset](https://github.com/opengeospatial/ogc_api_coverages/blob/master/standard/requirements/geodata-coverage/REQ_domainset-success.adoc).

How about Index3D and Index4D? 3D adds `k` and `4D` adds `m`, but if you have height + time, or only height, or only time, would one be preferably associated with a specific dimension? If not, if you have both would height be `k` and time `m`?

**Comment by pebau** [comment]
There was a corrigendum to all CIS/WCS for allowing free axis choice, which became necessary after a unilateral move of OGP. Unfortunately, OGC is grossly lagging behind publishing those (more than 1 year), maybe Stephan knows more about the status.

Index4D + height + time = 6D, believe this is not what you want.

**Comment by jerstlouis** [comment]
@pebau Are you saying that those 4D of _Index4D_ never include height and/or time?
Can't time and/or height also be indexed just like the spatial dimensions?
Thanks!

**Comment by pebau** [comment]
yes, surely you can map time etc to Index, such as using Index4D and calling axes "Lat Long h time"  - you just have no uom and only integer coordinates.
